docs/event-schema/index.md

```diff
@@ -0,0 +1,9 @@
+# Event schema
```

Comment:
I would like to keep this in for as long as we don't have the schema properly integrated in gatsby. But the end goal would definetly be to move this to develop.sentry.dev